tests/test_lightblue.py

```diff
@@ -1393,34 +1591,92 @@ def test_images_with_included_newer_srpm(
         repositories = {
             repo["repository"]: repo for repo in
             self.fake_repositories_with_content_sets}
-        cont_images.return_value = (
-            self.fake_container_images +
-            self.fake_container_images_floating_tag)
         ret = lb.find_images_with_included_rpms(
             ["content-set-1", "content-set-2"], ["openssl-1.2.3-1"], repositories)
```

Comment:
As above, check the method definition to see what's wrong here.

Reply:
Resolving this as discussed!